### PR TITLE
Updated TodoTxtMac to version 2.1.1.

### PR DESCRIPTION
**Pull request by mjdescy**

### CODE REVIEW

File: Casks/todotxt.rb [hunk 1]
```diff
@@ -1,11 +1,9 @@
-cask 'todotxt' do
-  version '2.1.0'
-  sha256 '9f58cd655a4f8711ac6419038328a99a631944a17417b457254412996749a393'
+cask :v1 => 'todotxt' do
```
Comment (vitorgalvao) [inline Casks/todotxt.rb]: This addition is wrong, and needs to be removed. We haven’t used that for a while.

File: Casks/todotxt.rb [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
+  version '2.1.1'
+  sha256 '84a45343e9611a1a2e62c6ad4eaa45b9373e2a74490086af548da64ee5e86940'
 
-  # github.com/mjdescy/TodoTxtMac was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/todotxt.rb]: Don’t remove the comment. It’s there for a reason.

File: Casks/todotxt.rb [hunk 3]
```diff
@@ -1,11 +1,9 @@
...
   url "https://github.com/mjdescy/TodoTxtMac/releases/download/#{version}/TodoTxtMac.app.zip"
-  appcast 'https://github.com/mjdescy/TodoTxtMac/releases.atom',
```
Comment (vitorgalvao) [inline Casks/todotxt.rb]: Don’t remove these two lines. They’re there for a reason.

Why are you removing them in the first place?

File: Casks/todotxt.rb [hunk 4]
```diff
@@ -1,11 +1,9 @@
...
-          checkpoint: 'ef8102fe3bd7e8f151c407cf11250a8b942080a22e9effe1c229a47fad5fe222'
+  appcast 'https://github.com/mjdescy/TodoTxtMac/releases.atom'
```
Comment (vitorgalvao) [inline Casks/todotxt.rb]: This makes no sense, you’re removing mandatory stuff for no reason.

**Comment by mjdescy** [comment]
I apologize for all those errors. I see now that they were a result of a bad merge on my end. I never intended to undo the changes you pointed out; I wasn't aware that I was doing it. I will withdraw this request and create a corrected one. Sorry to waste your time.

**Comment by vitorgalvao** [comment]
No need, I can make the corrections. But I’d still like to know why they were done. Was something unclear in the documentation that made it seems these changes were expected?

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 77d7884cb789e05ba8fd8618ca8d4b13973861f8. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by mjdescy** [comment]
Vítor,

Thank you for straightening it out for me. The errors were all mine. I didn't review the docs well enough to understand the process had changed since the last time I made a contribution. I will try to do a lot better next time, and plan to use the cask-repair script.

**Comment by vitorgalvao** [comment]
Alright. Thank you again for the contribution. We’re here for any questions you may have.

**Pull request closed without merging**